common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -87,21 +86,127 @@ object WidthsByBreakpoint {
+
+  object BodyMedia extends ContentRelation {
+    val Inline = WidthsByBreakpoint(
+      mobile =          Some(445.px),
+      mobileLandscape = Some(605.px),
+      phablet =         Some(620.px)) // tablet, desktop, leftCol and wide are also 620px
+
+    val Supporting = WidthsByBreakpoint(
+      mobile =          Some(445.px),
+      mobileLandscape = Some(605.px),
+      phablet =         Some(620.px), // tablet is also 620px
+      desktop =         Some(300.px), // leftCol is also 300px
+      wide =            Some(380.px))
+
+    val Showcase = WidthsByBreakpoint(
+      mobile =          Some(445.px),
+      mobileLandscape = Some(605.px),
+      phablet =         Some(620.px), // tablet and desktop are also 620px
+      leftCol =         Some(780.px),
+      wide =            Some(860.px))
+
+    val Thumbnail = WidthsByBreakpoint(
+      mobile =          Some(120.px), // mobileLandscape and tablet are also 120px
+      tablet =          Some(140.px)) // desktop, leftCol and wide are also 140px
+  }
+
+  object MainMedia extends ContentRelation {
+    val Inline = WidthsByBreakpoint(
+      mobile =          Some(465.px),
+      mobileLandscape = Some(645.px),
+      phablet =         Some(620.px),
+      tablet =          Some(700.px),
+      desktop =         Some(620.px)) // leftCol and wide are also 620px
+
+    val Showcase = WidthsByBreakpoint(
+      mobile =          Some(465.px),
+      mobileLandscape = Some(645.px),
+      phablet =         Some(620.px),
+      tablet =          Some(700.px),
+      desktop =         Some(620.px),
+      leftCol =         Some(780.px),
+      wide =            Some(860.px))
+
+    val FeatureShowcase = WidthsByBreakpoint(
+      mobile =          Some(465.px),
+      mobileLandscape = Some(645.px),
+      phablet =         Some(725.px),
+      tablet =          Some(965.px),
+      desktop =         Some(1125.px),
+      leftCol =         Some(1140.px),
+      wide =            Some(1300.px))
+  }
+
+  object PictureMedia {
+    // PictureMedia does not support hinting/weighting, so does not extend ContentRelation.
+    val Inline = WidthsByBreakpoint(
+      mobile =          Some(465.px),
+      mobileLandscape = Some(645.px),
+      phablet =         Some(685.px),
+      tablet =          Some(700.px),
+      desktop =         Some(940.px)) // leftCol and wide are also 940px
+  }
+
+  object GalleryMedia {
+    val Inline = WidthsByBreakpoint(
+      mobile          = Some(445.px),
+      mobileLandscape = Some(610.px),
+      phablet =         Some(620.px),
+      tablet =          Some(700.px)) // desktop, leftCol, and wide are also 700px
+
+    val Lightbox = WidthsByBreakpoint(
+      mobile =          Some(465.px),
+      mobileLandscape = Some(645.px),
+      phablet =         Some(725.px),
+      tablet =          Some(965.px),
+      desktop =         Some(1065.px),
+      leftCol =         Some(1225.px),
+      wide =            Some(1920.px))
+  }
```

Comment:
Wowsa, a lot more variations of widths in Content. You might want to add Profiles for these if you've not ...